base/docs/basedocs.jl

```diff
@@ -2903,6 +2903,9 @@ UnionAll
 julia> typeof(Vector{Int})
 DataType
 ```
+
+
+See the manual section on [UnionAll Types](@ref).
```

Comment:
Can you move this sentence to before the *Examples* section? Also, I think it would be fine to just put it at the end of the existing paragraph, instead of creating a second paragraph.

Reply:
Sure, made the changes!